screens/Menu/NearbyInterestDetails.js

```diff
@@ -1,33 +1,151 @@
 import React from "react";
-import { View, Text, StyleSheet } from "react-native";
+import { View, Text, StyleSheet, Image, SafeAreaView } from "react-native";
 import { Icon, Button } from "native-base";
+import StarRating from "react-native-star-rating";
 
 
+/**
+ * US34 - As a user, I would like to see the nearest outdoor points of interest #14
+ * US35 - As a user, I would like to get the direction to the chosen nearest point of interest #15
+ * US36 - As a user, I would like to see a detailed description of the selected places #29
+ * 
+ * Description: This screen will presents the layout 
+ * for the display of the point of interests. The points 
+ * of interest information will be retrieved from the Google Places API.
+ * The screen is composed on a flatList to create a grid, and have 
+ * an pressable item for each points of interest
+ */
+
 /**Prop passed
  * @param  {} navigation props.navigation is the name of the object from Navigator library
  */
 
 function NearbyInterestDetails(props) {
+    /**
+     * navigation that serves to go back to previous screen
+     * @param  {} =>{props.navigation.goBack(
+     */
 
     const goBack = () => {
         props.navigation.goBack();
     }
 
-    const NearbyInterestDetailsScreen = true; 
+    const item = props.navigation.getParam("item", null);
+    const name = props.navigation.getParam("name", null);
+    const photoref = item.img;
+    const rating = item.rating;
+    const hours = item.open_hours;
```

Comment:
If `item` is null, the app will crash saying can't read property of null. Maybe add error handling here:

`const photoref = item ? item.img : null;`

Reply:
Yep, that can be done!